packages/cli/src/commands/capabilities/translations/create.ts

```diff
@@ -0,0 +1,77 @@
+import { CapabilityLocalization } from '@smartthings/core-sdk'
+
+import { APICommand, inputAndOutputItem } from '@smartthings/cli-lib'
+
+import { buildTableOutput } from '../translations'
+import { capabilityIdInputArgs, chooseCapability } from '../../capabilities'
+
+
+export default class CapabilityTranslationsCreateCommand extends APICommand {
+	static description = 'Create or update a capability translation'
```

Comment:
Should this descriptor just be `Create` and the one below be `Update`? They both say `Create or update`.

Reply:
Good catch...copied from the `upsert` command and forgot to change.